app/controllers/transcripts_controller.rb

```diff
@@ -31,6 +31,10 @@ def search
   # GET /transcripts/the-uid
   # GET /transcripts/the-uid.json
   def show
+    if logged_in_user.admin? || logged_in_user.content_editor?
```

Comment:
We already have the `before_action :set_transcript` that applies to this method; it should be fetching the transcript using TranscriptService. I'm assuming this is intended to get the admin-side preview working - is there anything special that this does that `set_transcript` doesn't? (Maybe we should roll them both together?)

Reply:
Hi @rtrvrtg, yes, the `TranscriptService#find_by_uid` only fetches the `published` transcripts, so we need to override the `@transcript` when we want to preview the unpublished transcripts.

Reply:
Fair enough. We could consolidate the access logic later on, but for just viewing the transcripts, it should be fine to override it in this one place.